src/main/java/org/gridsuite/study/server/service/StudyService.java

```diff
@@ -1840,6 +1839,12 @@ public void deleteNodes(UUID studyUuid, List<UUID> nodeIds, boolean deleteChildr
                 }
             });
         }
+    }
+
+    @Transactional
+    public void deleteNodes(UUID studyUuid, List<UUID> nodeIds, boolean deleteChildren, String userId) {
+        removeNodesFromAliases(studyUuid, nodeIds);
```

Comment:
I cant see why we need this, because alias update is already made when we stash a node.

Reply:
If we just call manually the endpoint that delete nodes (with a curl ...) without having called the endpoint that stash nodes ...